pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1685,7 +1684,7 @@ func TestTCPDirectionWithPreexistingConnection(t *testing.T) {
 	// create an initial client connection to the server
 	c, err := net.DialTimeout("tcp", server.address, 5*time.Second)
 	require.NoError(t, err)
-	defer c.Close()
+	t.Cleanup(func() { c.Close() })
```

Comment:
nit: equivalent with "less code"
```suggestion
	t.Cleanup(c.Close)
```

Reply:
`c.Close()` returns `error` so can't pass it to `t.Cleanup`